charts/gardener-extension-provider-azure/values.yaml

```
@@ -10,10 +10,10 @@ maxSurge: 50%
 resources: {}
 vpa:
   enabled: true
-# resourcePolicy:
-#   minAllowed:
-#     cpu: 100m
-#     memory: 128Mi
+  resourcePolicy:
+    minAllowed:
+      cpu: 10m
+      memory: 120Mi
```

I would rather stick to lower minAllowed wrt the memory. For example the values from os-gardenlinux extension - https://github.com/gardener/gardener-extension-os-gardenlinux/blob/94ea26e9be17b375e70e24bed432e78197d7a330/charts/gardener-extension-os-gardenlinux/values.yaml#L12-L13

```suggestion
      cpu: 50m
      memory: 64Mi
```

---

`make generate` is needed :(